src/tests/routes/protectedRoute.test.js

```diff
@@ -5,12 +5,12 @@ import mockAxios from 'axios';
 import { createMemoryHistory } from 'history';
```

Comment:
@rafayet-monon Should we also rename this file to `routeAuthentication.test.js`

Reply:
@carryall fixed in [5bc9b52](https://github.com/rafayet-monon/surveyor/pull/27/commits/5bc9b52ed6a1bbc79d142f22a00428ea1335472e)